src/file.ts

```diff
@@ -52,8 +52,8 @@ export async function getEntryLink(this: SkynetClient, userID: string, path: str
   validateString("path", path, "parameter");
 
   const dataKey = deriveDiscoverableTweak(path);
-  const opts = defaultGetEntryOptions;
-  opts.hashedDataKeyHex = true; // Do not hash the tweak anymore.
+  // Do not hash the tweak anymore.
+  const opts = { ...defaultGetEntryOptions, hashedDataKeyHex: true };
```

Comment:
I'm kind of surprised this is the only occurrence of this type of bug. Especially since we use the defaultXOptions all over the place, did you verify that?

Reply:
Ok I checked and I think it's missing here as well https://github.com/SkynetLabs/skynet-js/blob/master/src/mysky/index.ts#L308

Reply:
good catch @peterjan, I went in and checked codebase myself against that bug too and it looks like all of it